src/pytorch_lightning/core/datamodule.py

```diff
@@ -75,7 +77,7 @@ def add_argparse_args(cls, parent_parser: ArgumentParser, **kwargs) -> ArgumentP
         return add_argparse_args(cls, parent_parser, **kwargs)
 
     @classmethod
-    def from_argparse_args(cls, args: Union[Namespace, ArgumentParser], **kwargs):
+    def from_argparse_args(cls, args: Union[Namespace, ArgumentParser], **kwargs: Any) -> ArgumentParser:
```

Comment:
this isn't correct; I am trying to see if I can force CI code-check to error and provide a hint for the correct return type